test.js

```diff
@@ -0,0 +1,381 @@
+import test from 'ava';
+import 'babel-core/register';
```

Comment:
I _guess_ I could configure it in the package.json instead, if you think that's nicer?